system/cfme-setup.sh

```diff
@@ -31,6 +31,17 @@ cat <<'EOF' > /etc/httpd/conf.d/ssl.conf
 # upgraded.
 EOF
 
+# Incorporate the iptable updates
+if [[ -f /etc/sysconfig/iptables.miq ]]; then
+  /sbin/iptables-save > /etc/sysconfig/iptables.SYSTEM
+  /sbin/iptables-restore /etc/sysconfig/iptables.miq
+  /sbin/service iptables save
+  # Expect no more than the four datestamp differences.
+  if [[ $(/usr/bin/diff -Ew -U 0 /etc/sysconfig/iptables /etc/sysconfig/iptables.miq  | /bin/grep "^@@" | /usr/bin/wc -l) -gt 4 ]]; then
+    /bin/echo "FAILURE Incorporating the iptable updates."
```

Comment:
@jrafanie  I don't want to exit and abort the entire installation if we detect an issue. I only want to log the issue.